pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/LoadBalancerTest.java

```diff
@@ -321,8 +322,12 @@ public void testBrokerRanking() throws Exception {
         }
 
         for (int i = 0; i < BROKER_COUNT; i++) {
-            Method updateRanking = Whitebox.getMethod(SimpleLoadManagerImpl.class, "updateRanking");
-            updateRanking.invoke(pulsarServices[i].getLoadManager().get());
+            Method method = Whitebox.getMethod(SimpleLoadManagerImpl.class, "getUpdateRankingHandle");
+            Object invoke = method.invoke(pulsarServices[i].getLoadManager().get());
```

Comment:
Sorry, I could have suggested it before...
We recently started to use Awaiatility for this kind of waits

What about rewriting the loop this way?


```
Awaiatility.await().until( () -> {
 Object invoke = method.invoke(pulsarServices[i].getLoadManager().get());
 return invoke != null && invoke.isDone();
});
```

Reply:
Yes, I will update this. And try to keep the step with community and keep learning from you, from who giving me suggestions and ideas. Thanks very much for reviewing.